jans-linux-setup/jans_setup/setup_app/installers/shibboleth.py

```
@@ -153,60 +153,49 @@ def generate_signing_keys(self):
     def configure_idp_properties(self):
         idp_properties_template = os.path.join(self.template_dir, 'idp.properties')
         idp_properties_output = os.path.join(self.shibboleth_home, 'conf', 'idp.properties')
-        
+
         self.renderTemplateInOut(idp_properties_template, self.template_dir, 
                                   os.path.join(self.shibboleth_home, 'conf'))
 
     def configure_jans_authentication(self):
         jans_properties_template = os.path.join(self.template_dir, 'jans.properties')
         jans_properties_output = os.path.join(self.shibboleth_home, 'conf', 'jans.properties')
```

Ruff (0.14.14)

[error] Local variable `jans_properties_output` is assigned to but never used

Remove assignment to unused variable `jans_properties_output`

(F841)